pkg/processor/build/builder.go

```diff
@@ -524,6 +553,17 @@ func (b *Builder) copyObjectsToStagingDir() error {
 	return nil
 }
 
+func (b *Builder) cleanupTempDir() error {
+	err := os.RemoveAll(b.tempDir)
+	if err != nil {
+		return errors.Wrapf(err, "Failed to clean up temp dir %s", b.tempDir)
```

Comment:
Please arrive at a consistent form to refer to these directories across all comments, user-facing errors and log messages. `temporary dir` is probably most consistent with the already-existing `staging dir`. For variable names, `tempDir` is fine - there's no need to align that too.